Maybe not suitable on arm architecture in production when deployment is build on amd64

I am running into issues with a current Plone6 Volto deployment created by cookieplone 0.8.1 when using the `make stack-deploy` command and check with `make stack-status`.

I am guessing it is maybe related to platform mismatch issues.

The development machine is on Ubuntu 24.04 LTS amd64 (`uname -p` -> `x86_64`) but the Hetzner Cloud server is running on Ubuntu 24.04 LTS arm64 (`uname -p` -> `aarch64`). `aarch64` or `arm64` denotes an ARM64 architecture

The Production Target is using docker engine 27.5.1
The Dev machine version 27.5.0

The later `make create-site` fails with a 

```
...
+ docker --context prod exec ./docker-entrypoint.sh create-site
Error response from daemon: No such container: docker-entrypoint.sh
make: *** [Makefile:98: stack-create-site] Error 1
```

I had a similar error on the same target architecture but a different project. I did not get it to work then under cookieplone 0.8.0
see: https://community.plone.org/t/unclear-instructions-how-to-check-if-docker-images-build-is-working-stack-create-site-error/19394
Instead of the older issue currently all my GitHub actions run fine with no errors an my own images are build well on ghcr.io

### Details

`make stack-deploy` output:

```
++ tput setaf 2
++ tput sgr0
+ echo '==> Stack xxx-dev-acsr-de: Deploy to context prod '
==> Stack xxx-dev-acsr-de: Deploy to context prod 
+ docker --context prod stack deploy --with-registry-auth -c stacks/xxx.dev.acsr.de.yml xxx-dev-acsr-de
Since --detach=false was not specified, tasks will be created in the background.
In a future release, --detach=false will become the default.
Updating service xxx-dev-acsr-de_traefik (id: zhzvszpg01zht6xvpxp0la95m)
Updating service xxx-dev-acsr-de_purger (id: qgw7m7ibnxog23qz2fjzq1q44)
Updating service xxx-dev-acsr-de_varnish (id: xvjskwlbx3altgr2c9zq8hdcq)
image ghcr.io/acsr-industrialdesign/xxx-volto-varnish:latest could not be accessed on a registry to record
its digest. Each node will access ghcr.io/acsr-industrialdesign/xxx-volto-varnish:latest independently,
possibly leading to different nodes running different
versions of the image.

Updating service xxx-dev-acsr-de_frontend (id: mvqeedogmmigmxabfr943rzvk)
image ghcr.io/acsr-industrialdesign/xxx-volto-frontend:latest could not be accessed on a registry to record
its digest. Each node will access ghcr.io/acsr-industrialdesign/xxx-volto-frontend:latest independently,
possibly leading to different nodes running different
versions of the image.

Updating service xxx-dev-acsr-de_backend (id: z5rygpslgdrjrpkwnjfqw2zyh)
image ghcr.io/acsr-industrialdesign/xxx-volto-backend:latest could not be accessed on a registry to record
its digest. Each node will access ghcr.io/acsr-industrialdesign/xxx-volto-backend:latest independently,
possibly leading to different nodes running different
versions of the image.

Updating service xxx-dev-acsr-de_db (id: puipb0tcckdfd26pudhl8bdz2)
```

`make stack-status` output (error truncated)

```
++ tput setaf 2
++ tput sgr0
+ echo '==> Stack xxx-dev-acsr-de: Status in context prod '
==> Stack xxx-dev-acsr-de: Status in context prod 
+ docker --context prod stack ps -f desired-state=running xxx-dev-acsr-de
ID             NAME                        IMAGE                                      NODE      DESIRED STATE   CURRENT STATE         ERROR                              PORTS
mss145a5ck7r   xxx-dev-acsr-de_db.1        postgres:14.15                             xxx       Running         Running 2 hours ago                                      
h7eov4scvoer   xxx-dev-acsr-de_purger.1    ghcr.io/kitconcept/cluster-purger:latest             Running         Pending 2 hours ago   "no suitable node (unsupported…"   
mzvre3jg2os8   xxx-dev-acsr-de_purger.2    ghcr.io/kitconcept/cluster-purger:latest             Running         Pending 2 hours ago   "no suitable node (unsupported…"   
rudokfvmzzr2   xxx-dev-acsr-de_traefik.1   traefik:v2.11                              xxx       Running         Running 2 hours ago                 
```
The error is expected to be `no suitable node (unsupported platform on 1 node)`

### Question?
How can I further debug if I need a different arm64 image and or how to build it from scratch?

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -74,7 +74,7 @@ jobs:
       - name: Build and push
         uses: docker/build-push-action@v5
         with:
-          platforms: linux/amd64
+          platforms: linux/amd64,linux/arm64
           context: .
           file: Dockerfile
           push: ${{ github.event_name != 'pull_request' }}

